charts/seed-controlplane/charts/vpn-seed-server/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Don't introduce new Helm charts, please. Use the [component structure](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/component/interfaces.go) (like for example in https://github.com/gardener/gardener/tree/master/pkg/operation/botanist/component/metricsserver).

Reply:
We adapted it accordingly.